src/util_impl_linux.h

```diff
@@ -440,12 +440,16 @@ class CpuInfoLinux : public CpuInfo {
 
     FILE *file = fopen(path_midr_el1, "r");
     if (file == nullptr) {
-      throw Error(ERR_INTERNAL);
+      // sysfs is unaccessible for AWS Lambdas
+      fprintf(stderr, "%s, %d: Can't open MIDR_EL1 sysfs entry\n", __FILE__, __LINE__);
```

Comment:
I suppose `printf` is not needed here, because this code is used in production where such messages are not allowed

Reply:
@sandye51 
Thank you for the comment. 
How about printfing only when the environment variable XBYAK_AARCH64_VERBOSE is defined?

Reply:
@sandye51 hmm, isn't it the same behavior as one allowed by XBYAK_AARCH64_ERROR_
https://github.com/fujitsu/xbyak_aarch64/blob/f43f7e872e9af1555a8c73602cae2c555a2b4919/src/util_impl_linux.h#L51